src/app/models.py

```diff
@@ -11,18 +11,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
 from __future__ import unicode_literals
-
 from django.db import models
 from datetime import datetime
 from django import forms
 from django.contrib.auth.models import User
 
-
 class UserID(models.Model):
     user = models.OneToOneField(User)
     organisation = models.CharField("Organisation",max_length=64, null=False, blank=False)
     lastlogin = models.DateField("Last Login",default=datetime.now,blank=True)
     def __str__(self):
         return self.user.username
+
+class LicenseRequest(models.Model):
+    fullname = models.CharField(max_length=70)
+    shortIdentifier = models.CharField(max_length=25)
+    submissionDatetime = models.DateTimeField(auto_now_add=True)
+    userEmail = models.EmailField(max_length=35)
+    xml = models.TextField()
+
+    def __unicode__(self):
+        return "%s %s %s %s %s" % (self.fullname, self.shortIdentifier, self.submissionDatetime, self.userEmail, self.xml)
```

Comment:
You can only return the unique identifier for this.I(Like fullname or shortIdentifier). It shows like this in the admin window.
![screenshot-2018-7-1 select licenserequest to change django site admin](https://user-images.githubusercontent.com/17159160/42139637-dc6eab7e-7d5f-11e8-94a7-872f1bf0ecef.png)

Doesn't look good.

Reply:
Done. Now it just shows the license fullname.